core/services/inventory.py

```diff
@@ -52,6 +53,29 @@ def createInventoryItem(self,inventoryItem) -> None:
                 "purchasing_price":new_inventoryItem.purchasing_price,\
                 "selling_price":new_inventoryItem.sales_service_item.price})
         return 
+    
+    def createMultipleInventoryItem(self,inventoryItems) -> None:
+        try:
+            for inventoryItem in inventoryItems:
+                if type(inventoryItem) is not dict:
```

Comment:
Umm... Bro you do know you can reuse the `def createInventoryItem(self,inventoryItem):` function right? XD